cg/meta/demultiplex/housekeeper_storage_functions.py

```diff
@@ -190,7 +190,7 @@ def add_sample_sheet_path_to_housekeeper(
     """Add sample sheet path to Housekeeper."""
 
     try:
-        sample_sheet_file_path: Path = get_sample_sheet_path(
+        sample_sheet_file_path: Path = get_sample_sheet_path_from_flow_cell_dir(
```

Comment:
Had to rename this function to not clash with the new one

Reply:
This should fetch from housekeeper not from the flow cell directory.